pkgs/applications/office/jabref/default.nix

```diff
@@ -146,6 +146,7 @@ stdenv.mkDerivation rec {
     DEFAULT_JVM_OPTS=$(sed -n -E "s/^DEFAULT_JVM_OPTS='(.*)'$/\1/p" $out/bin/JabRef | sed -e "s|\$APP_HOME|$out|g" -e 's/"//g')
     rm $out/bin/*
 
+    runHook gappsWrapperArgsHook
```

Comment:
This is odd, you probably want to disable gapps wrapping and add the flags to makeWrapper to also prevent double wrapping

Reply:
It's what I currently do but in installPhase the wrapper args are not complete. And yes, this is odd...

Reply:
I don't know why gappsWrapperArgsHook is run in preFixup. Maybe it should run in preInstall instead?

Reply:
Can we just move the wrapping to postFixup? This would way easier than changing the wrapper itself.

Reply:
Yes, it's easier. Still odd but easy.

Reply:
Done.